test/plugins/builds.test.js

```diff
@@ -2728,41 +2726,14 @@ describe.only('build plugin test', () => {
             };
         });
 
-        it('returns 200 when updating the code/endTime', () => {
-            stepFactoryMock.get.withArgs({ buildId: id, name: step }).resolves(null);
-
-            return server.inject(options).then((reply) => {
-                assert.equal(reply.statusCode, 200);
-                assert.deepProperty(reply.result, 'name', 'test');
-                assert.deepProperty(reply.result, 'code', 0);
-                assert.deepProperty(reply.result, 'endTime', options.payload.endTime);
-                assert.notDeepProperty(reply.result, 'startTime');
-            });
-        });
-
-        it('returns 200 when updating the code/endTime when the step model exists', () => {
+        it('returns 200 when updating the code/endTime when the step model exists', () =>
             server.inject(options).then((reply) => {
```

Comment:
Do we need to add return statement?

Reply:
This function is written in arrow function without braces.

Reply:
I see.